CMakeLists.txt

```diff
@@ -4714,8 +4717,24 @@ endif()
 
 # rendergraph
 add_subdirectory(src/rendergraph)
+target_compile_definitions(
+  rendergraph_gl
+  PUBLIC $<$<CONFIG:Debug>:MIXXX_DEBUG_ASSERTIONS_ENABLED>
+)
 target_link_libraries(mixxx-lib PUBLIC rendergraph_gl)
 target_compile_definitions(mixxx-lib PUBLIC rendergraph=rendergraph_gl)
+target_compile_definitions(mixxx-lib PRIVATE allshader=allshader_gl)
+target_compile_definitions(mixxx-test PRIVATE allshader=allshader_gl)
```

Comment:
missing if(BUILD_TESTING)/endif()

```
CMake Error at CMakeLists.txt:4727 (target_compile_definitions):
  Cannot specify compile definitions for target "mixxx-test" which is not
  built by this project.

```

Reply:
Is there a CI run for this or how does this happen?

Reply:
checked with live ebuild from gentoo. But I think base/test and SHARED_LIBS (https://github.com/mixxxdj/mixxx/pull/14021#issuecomment-2630601087) should be integrated in CI.

Reply:
@acolombier is it feasible for you to look into this?

Reply:
> how can I reproduce this?

BUILD_TESTING=OFF

Reply:
This should be fixed now. Thanks for reporting!